src/IcedTasks/CancellablePoolingValueTask.fs

```diff
@@ -108,7 +108,7 @@ module CancellablePoolingValueTasks =
                         let mutable __stack_exn = null
 
                         try
-                            let __stack_code_fin = code.Invoke(&sm)
+                            let __stack_code_fin = (yieldOnBindLimit code).Invoke(&sm)
 
                             if __stack_code_fin then
                                 MethodBuilder.SetResult(&sm.Data.MethodBuilder, sm.Data.Result)
```

Comment:
SetResult and SetException will stack dive on old .Net.
So this also needs to be wrapped with `yieldOnBindLimit`